src/foam/nanos/jetty/HttpServer.js

```diff
@@ -293,6 +301,18 @@ foam.CLASS({
   
       }
       `
+    },
+    {
+      name: 'containsHostDomain',
+      type: 'Boolean',
+      documentation: `Returns true if given domain is contained in server's host domains.`,
+      args: [
+        { name: 'domain', javaType: 'String' }
+      ],
+      javaCode: `
+        int a = Arrays.binarySearch(getHostDomains(), domain);
```

Comment:
Assume this was for debugging purposes. 
```suggestion
        return Arrays.binarySearch(getHostDomains(), domain) >= 0;
```